ui-react/packages/atlasmap-ui/src/AtlasmapUI/AtlasmapCanvasView/AtlasmapCanvasViewMappings.tsx

```diff
@@ -5,6 +5,7 @@ import { useAtlasmapUI } from '../AtlasmapUIProvider';
 
 export interface IAtlasmapCanvasViewMappingsProps {
   onAddToMapping: (elementId: ElementId, mappingId: string) => void;
+  onCreateMapping: (sourceId: ElementId, targetId: ElementId) => void;
```

Comment:
This isn’t used, is it?

Reply:
Actually no - seemed like reasonable functionality to make available.

Reply:
To create a mapping using a “new mapping” target in the mapping column we’ll need to either allow for creating incomplete mappings or figure out how to delay the mapping creation until we have at least a source and a target.

If incomplete mappings are possible then it’s easy, but we’ll need a different callback since we’ll have a single field to deal with.

Does it make sense?

Reply:
true - new PR on its way